core/src/com/fourinachamber/fortyfive/game/controller/EncounterContext.kt

```diff
@@ -0,0 +1,64 @@
+package com.fourinachamber.fortyfive.game.controller
+
+import onj.value.OnjNamedObject
+
+interface EncounterContext {
+
+    val encounterIndex: Int
+
+    val forwardToScreen: String
+
+    val forceCards: List<String>?
+        get() = null
+
+    fun completed()
+}
+
+
+sealed class RevolverRotation {
```

Comment:
extract into own file or move to another class, as it doesn't make any sense in a file called "EncounterContext"